lib/util/versioning.js

```diff
@@ -17,6 +17,8 @@ if (process.env.NODE_PRE_GYP_ABI_CROSSWALK) {
     abi_crosswalk = require('./abi_crosswalk.json');
 }
 
+var jsEngine = process.jsEngine ? '-' + process.jsEngine : '';
```

Comment:
>jsEngine [](start = 4, length = 8)

as mentioned in my previous comments, i would do explicit check for `chakracore` here.